### ISSUE TITLE
No free -h on CentOS 6

### ISSUE THREAD

**Issue by mr119** [masonr/yet-another-bench-script#8]
Great script! Forked it for our internal use.

The script uses `free -h` to get RAM information, however on **CentOS 6** surprisingly `free -h` isn't available, so might have to update with free and convert the response to human-readable numbers.

**Comment by masonr** [comment]
Hey there, thanks for the bug report! I'll look into this and see if I can replicate. Just using `free` without options and converting to a human-readable format within the script is easily doable since the disk speed results already do something similar. Let me test and write up a fix for this one. Thanks again!

**Comment by masonr** [comment]
Hello there @mr119!

Please give the following a test and make sure the fix I implemented works well. I confirmed using a CentOS 6 Docker container that the 'free -h' problem existed and that this fix worked. Please make sure this is the case on your end as well. 

```
curl -s https://raw.githubusercontent.com/masonr/yet-another-bench-script/testing/yabs.sh | bash -s -- -dig
```

Thanks!

**Comment by masonr** [comment]
Fixed in 203c9a0
